docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1507,6 +1507,51 @@ As the core components use emotion as a styled engine, the props used by emotion
   }
   ```
 
+# createGenerateClassName
```

Comment:
h1 is wrong. But I also wonder about the heading structure. What if we grouped all changes related to the package under a `### @material-ui/styles`

Reply:
Should we maybe group them under `### @material-ui/core/styles` as the breaking changes are related to this specific import path?

Reply:
+1, this sounds better. My proposal was taking the point of view of a v5 user, yours of a v4 user, which where most readers will be.